packages/webview_flutter/webview_flutter_wkwebview/README.md

```diff
@@ -34,12 +34,12 @@ Then you will have access to the native class `FWFWebViewFlutterWKWebViewExterna
 This package uses [pigeon][3] to generate the communication layer between Flutter and the host
```

Comment:
We should really consider moving this to a CONTRIBUTING.md in the package, instead of it being in the client-facing README.

Reply:
I'll create a separate PR for this